### PR TITLE
Rk wip frd automation

### PR DESCRIPTION
**Pull request by rishav-code**
2nd sheet completed having a pull request to review the code

### CODE REVIEW

File: src/main/java/com/frd/automation/pages/DashboardHomepage.java [hunk 1]
```diff
@@ -49,9 +47,10 @@ public DashboardHomepage(WebDriver driver) throws IOException {
      */
     public void validateDashboardHomePageLanding() {
         String homePageTitle = driver.getTitle();
-        Assert.assertEquals(homePageTitle, validationData.getValidationData("dashboard.homepage.title"));
         String homePageUrl = driver.getCurrentUrl();
-        Assert.assertEquals(homePageUrl, validationData.getValidationData("dashboard.homepage.url"));
+        validate.verifyTestData(homePageTitle, validationData.getValidationData("dashboard.homepage.title"), "Verifying the dashboard Title");
+        validate.verifyTestData(homePageUrl, validationData.getValidationData("dashboard.homepage.url"), "Verifying the dashboard Url");
+
```
Comment (rajmyil) [inline src/main/java/com/frd/automation/pages/DashboardHomepage.java:53]: why this space?

File: src/main/resources/locators/dashboard.properties [hunk 2]
```diff
@@ -16,6 +16,7 @@ dashboard.select.browser=browsers
 dashboard.testcase.name.txt=(//td[contains(@class,'ng-binding')])[1]
 dashboard.duration.txt=(//td[@class='ng-binding'])[2]
 dashboard.status.txt=//span[@class='label-pass ng-scope']
+dashboard.windows.sign=(//i[contains(@aria-hidden,'true')])[1]
```
Comment (rajmyil) [inline src/main/resources/locators/dashboard.properties:20]: why so many blank lines?

File: src/main/java/com/frd/automation/constants/Constants.java [hunk 3]
```diff
@@ -53,10 +53,9 @@
     public final static String TESTDATA_FILE= TESTDATA_HOME + "validationdata.properties";
     public final static String CHROME_PATH = USER_HOME + "drivers" + File.separator + "chromedriver.exe";
     public final static String CONFIG_FILE = RESOURCES_HOME + File.separator + "config.properties";
-    public final static String EXCEL_FILE = RESOURCES_HOME + File.separator + "TestReport-MakemytripSeatSelection-WEB-08-27-2021.xlsx";
+    public final static String ORIGINAL_EXCEL_FILE = RESOURCES_HOME + File.separator + "TestReport-MakemytripSeatSelection-WEB-08-27-2021.xlsx";
```
Comment (arunatm2017) [inline src/main/java/com/frd/automation/constants/Constants.java:56]: Hardcoded excel file.

File: src/main/java/com/frd/automation/base/TestBase.java [hunk 4]
```diff
@@ -62,9 +63,11 @@ public void launchBrowser() throws Exception {
             options.setExperimentalOption("prefs", prefs);
             WebDriverManager.chromedriver().setup();
             driver = new ChromeDriver(options);
-//            System.setProperty("webdriver.chrome.driver", directoryPath.CHROME_PATH);
```
[the remark this answers is not in the document]
Comment (rishav-code) [reply src/main/java/com/frd/automation/base/TestBase.java:65]: ok

**Pull request merged**